lib/tasks/all_incomplete.rake

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+namespace :update_all_stops do
+  desc "Mark all stops as incomplete"
+  task :all_incomplete do
```

Comment:
Probably need `:environment` here, right?

Reply:
Good call. We're using `uncomplete_all`. Unless anyone has any other ideas.